rules/REQUEST-901-INITIALIZATION.conf

```diff
@@ -8,6 +8,31 @@
 # ---------------------------------------------------------------
 
 #
+# -=[ Initialization / General Setup ]=-
```

Comment:
If possible could you add the difference between 901 and setup.conf to the documentation?